app/client/src/modules/ui-builder/ui/wds/WDSCustomWidget/widget/index.tsx

```
@@ -0,0 +1,121 @@
+  }
+
+  // TODO: Fix this the next time the file is edited
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  static getMetaPropertiesMap(): Record<string, any> {
+    return {
+      model: undefined,
+    };
+  }
+
+  execute = (eventName: string, contextObj: Record<string, unknown>) => {
+    if (this.props.hasOwnProperty(eventName)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)